src/pages/Upload.tsx

```
@@ -66,17 +135,88 @@
         multiple
         fileList={fileList}
         beforeUpload={() => false}
-        onChange={info => setFileList(info.fileList)}
+        onChange={info => {
+          setFileList(info.fileList)
+          // 初始化新文件的别名（默认使用推导出的 repo 名）
+          setAliases(prev => {
+            const next = { ...prev }
+            for (const f of info.fileList) {
+              const repo = repoNameFromFile(f.name)
+              if (!next.hasOwnProperty(repo)) next[repo] = repo
```

ESLint

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

(no-prototype-builtins)